hello_world/Android.mk

```diff
@@ -0,0 +1,20 @@
+###################### optee-hello-world ######################
+LOCAL_PATH := $(call my-dir)
+
+CFG_TEEC_PUBLIC_INCLUDE = $(LOCAL_PATH)/../../optee_client/public
```

Comment:
optee_benchmark uses `TEEC_EXPORT`
xtest uses `OPTEE_CLIENT_EXPORT`
Maybe sync with one of these.

Reply:
done